traffic_portal/app/src/common/modules/form/deliveryService/FormDeliveryServiceController.js

```diff
@@ -119,6 +119,15 @@ var FormDeliveryServiceController = function(deliveryService, dsCurrent, origin,
     }
     $scope.validateTLS = validateTLS;
 
+    let getSteeringTargets = function() {
```

Comment:
2 Questions: a) why the `let varName=  functionExpression` instead of `function functionName ... `?

And also, why not make this a bit cleaner by making function `async"?